readme.md

```diff
@@ -0,0 +1,30 @@
+#About the project
+*For read SRS-file [click](./documentation/SRS.md)*
+
+Need create a web service (RESTful) for **CRUD operations**. 
+One should be able to deploy the web service on **Gunicorn** 
+using the command line. All functions/methods on all levels 
+should include unit tests. Debug information should be displayed 
+at the debugging level in the console and in a separate file. 
+Classes and functions/methods must have docstrings comments.
+
+Need create a simple web application for managing departments and employees. 
+The web application should use the aforementioned web service for storing 
+data and reading them from the database. One should be able to deploy 
+the web application on **Gunicorn** using the command line. All public 
+functions/methods on all levels should include unit tests. 
+Debug information should be displayed at the debugging level 
+in the console and in a separate file. Classes and functions/methods 
+must have docstrings comments. 
```

Comment:
It looks like you just copied the task description. Could you rephrase these paragraphs so that they are more like a project description and not a task that needs to be implemented.

Reply:
Done